Python/Python.sublime-syntax

```diff
@@ -14,6 +14,7 @@ file_extensions:
   - gyp
   - gypi
   - Snakefile
+  - wscript
```

Comment:
my understanding from reading https://waf.io/book/#_using_several_scripts_and_folders is that `wscript` is not a file extension, but the full filename of a Waf project description file.